modules/reindex/src/main/java/org/elasticsearch/reindex/ReindexPlugin.java

```diff
@@ -85,9 +84,15 @@ public List<RestHandler> getRestHandlers(
 
     @Override
     public Collection<?> createComponents(PluginServices services) {
+        final Collection<Object> components = new ArrayList<>();
+        components.add(new ReindexSslConfig(services.environment().settings(), services.environment(), services.resourceWatcherService()));
+        components.add(new ReindexMetrics(services.telemetryProvider().getMeterRegistry()));
+        return (components);
```

Comment:
This can be simplified to:

```
return List.of(new ReindexSslConfig(services.environment().settings(), services.environment(), services.resourceWatcherService()), new ReindexMetrics(services.telemetryProvider().getMeterRegistry()));
```